### ISSUE TITLE
Cops Logo - Proposals

### ISSUE THREAD

**Issue by horus68** [seblucas/cops#246]
I'm not a master on logo design, but here are some proposals for a logo

option A
![cops_a](https://cloud.githubusercontent.com/assets/3055443/12355333/d66bdc50-bb93-11e5-80d3-12e12b89c90f.png)

option B
![cops_b](https://cloud.githubusercontent.com/assets/3055443/12355334/d670972c-bb93-11e5-96e0-39e5a15e8189.png)

option C
![cops_c](https://cloud.githubusercontent.com/assets/3055443/12355335/d67205c6-bb93-11e5-9b3a-7c362a0d9ea0.png)

option D
![cops_d](https://cloud.githubusercontent.com/assets/3055443/12355336/d67541f0-bb93-11e5-8d0b-96b8a97a7a83.png)

Note: I created this because I was not able to reopen previous issu:
Visual guidelines :) · Issue #185  - https://github.com/seblucas/cops/issues/185

**Comment by seblucas** [comment]
Disclaimer : I'm known to have a "curious taste" for any logo or design in general so please don't be shocked by any of my remarks ;)

I like the colours Option A (the light blue reminds me of the Kobo website). I'm still thinking if the name (`COPS`) should be included on the left or not.

The ratio will make it more difficult to be added in a `favicon.ico`, it would be easier if it had a 1:1 aspect ratio.

I also like Option C. I'll show your logo around and check if some of my coworkers have any advice.

**Comment by seblucas** [comment]
I forgot, thanks a lot for proposing some logos, I've been avoiding that for too long.

**Comment by horus68** [comment]
No problem... I'm not a designer and it was a little edit time for this.

Those icons are available from Flaticon as SVG files so you can make an improved version (I used Inkscape)
http://www.flaticon.com/free-icon/books-in-a-library_46828
http://www.flaticon.com/free-icon/open-magazine_88179

It would be useful to have a logo for COPS (and a related favicon because its also used as a bookmark image on feeds for ebook reader apps)

**Comment by scorphus** [comment]
Awesome! I kinda like option D. But maybe A's colors are better. Some details would vanish in a small version of it, like Sébastien pondered. I'll also show it to my colleagues and collect their thoughts on this. Thanks @horus68

**Comment by jmgant** [comment]
I like the tones of D. I use an e-ink browser and they render very nice in shades of grey.

**Comment by polytan02** [comment]
I agree with @jmgant : it needs to be nice with e-ink reader due to its purpose :)

**Comment by horus68** [comment]
@seblucas maybe you can revisit this?
- The actual favicon has an issue with translaparency (its black) so the option is to fix it or to make a new one from the "logo" image.
- A logo for Cops would be useful as a brand when you publish online as in https://hub.docker.com/r/lsioarmhf/cops/
- and maybe because a logo makes software to look more "pro" :)
